ocs_ci/ocs/md_blow.py

```diff
@@ -0,0 +1,155 @@
+import logging
+import threading
+from time import sleep
+
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.resources.pod import get_noobaa_core_pod, get_noobaa_db_pod
+from ocs_ci.ocs.resources.mcg import MCG
+from ocs_ci.utility.utils import run_cmd
+from ocs_ci.helpers import helpers
+
+
+logger = logging.getLogger(__name__)
+
+
+class MdBlow(object):
+    """
+    Upload s3 objects directly into database using MD_Blow script present in noobaa-core pod.
+    """
+
+    def __init__(self):
+        """
+        Assign default values to its parameter and patch required lines in core pod
+        """
+        self.db_percentage = None
+        self.obj_count = 1000
+        self.concurrency = 50
+        self.chunks = 1
+        self.chunk_size = 200
+        mcg_obj = MCG()
+        self.creds = mcg_obj.get_noobaa_admin_credentials_from_secret()
+        self.email = self.creds["email"]
+        self.password = self.creds["password"]
+        self.noobaa_core_pod = get_noobaa_core_pod()
+        self.noobaa_db_pod = get_noobaa_db_pod()
+        namespace = constants.OPENSHIFT_STORAGE_NAMESPACE
+        storage_cluster = constants.DEFAULT_STORAGE_CLUSTER
+
+        ptch = '{{"spec": {{"resources": {{"noobaa-core": {{"limits": {{"cpu": "6","memory": "4Gi"}}}}}}}}}}'
+        ptch_cmd = (
+            f"oc patch storagecluster {storage_cluster} "
+            f"-n {namespace}  --type merge --patch '{ptch}'"
+        )
+        run_cmd(ptch_cmd)
+        logger.info("Wait for noobaa-core pod move to Running state")
+        helpers.wait_for_resource_state(
+            self.noobaa_core_pod, state=constants.STATUS_RUNNING, timeout=300
+        )
```

Comment:
We should undo this optimization once we're done running the md_blow script. 

One safe approach to do so would be to serve the instance of this class via a fixture, and then add the necessary cleanup at its teardown.

Another could be to wrap `upload_obj_using_md_blow` in a method that would call it inside a try-finally block like this:
```
def upload_obj_using_md_blow(...):
    # Fetch default cpu and mem values to local variables
    ...
    ...
    # Patch high cpu and mem values to optimization
   ...
   ...
    try:
        self._actual_upload_obj_using_md_blow(...):
    finally:
        # Patch the original cpg and mem values
```